packaging/rpm/microshift-images.spec

```diff
@@ -0,0 +1,62 @@
+%global imageStore %{buildroot}/var/lib/microshift/images
+
+Name: microshift-images
+Version: 1
+Release: 1
+# this can be %{timestamp}.git%{short_hash} later for continous main builds
+Summary: Create custom container storage
+License: ASL 2.0
+URL: https://github.com/redhat-et/microshift
+
+BuildRequires:  podman
+BuildRequires: crio
+
+
+%description
+This rpm creates a custom RO container storage and pull images and add path to additional container image stores.
+
+%prep
+
+if [ -d  %{imageStore} ] 
+then
+   sudo rm -rf  %{imageStore}
+fi
+
+
+%build
+
+
+%install
+
+mkdir -p %{imageStore}
+
+declare -a ListOfImages=("quay.io/openshift/okd-content@sha256:27f7918b5f0444e278118b2ee054f5b6fadfc4005cf91cb78106c3f5e1833edd" \
```

Comment:
Instead of hardcoding image references here, could we extract them from the release metadata, e.g. [ ](https://github.com/redhat-et/microshift/blob/main/pkg/release/release_amd64.go)?
This is so it's a) guaranteed to be in sync when we rebase and b) it also works for non-amd64?

Reply:
It doesn't support non-amd64. Since I tested the rpmbuild on F35 (only amd64 devices), `podman pull` command implicitly pulls images for the respective processor type.